modules/ROOT/partials/configuration/a11ychecker_filter_issue.adoc

```diff
@@ -2,7 +2,7 @@
 
```

Comment:
Sorry I missed this as well, but this needs an id added.

- [ ] Add ids to the options for a11ychecker

Reply:
Done